examples/shirt-shop/app/.well-known/vercel/flags/route.ts

```diff
@@ -10,7 +9,8 @@ export async function GET(request: NextRequest) {
   const access = await verifyAccess(request.headers.get('Authorization'));
   if (!access) return NextResponse.json(null, { status: 401 });
 
-  // Forward info from Flags in Code
-  const providerData = await getProviderData(flags);
+  const providerData = await getProviderData(
```

Comment:
The changes to the example need to be removed before we can consider merging this

Reply:
My bad. Reverted those now.